css/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+	background-image: url('../images/hk.jpg');
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover; 
+}
+
+header {
+	font-size: 60px;
+	text-align: center;
+	font-family: 'Myriad Pro', 'Myriad', helvetica, arial, sans-serif;
+	text-shadow: 5px 5px 5px #292929;
+	/*letter-spacing: -7px;*/
+	color: white;
```

Comment:
This works as a fallback in case `-webkit-text-stroke` isn't implemented on the browser. Oh the joys of browser differences, nice!